app/views/middleware_messaging/_main.html.haml

```diff
@@ -1,4 +1,3 @@
-= render :partial => "layouts/flash_msg"
 .row
```

Comment:
Is this change OK? You're removing the flash partial and not doing anything else.